docs/sources/mimir/release-notes/v2.13.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Grafana Mimir version 2.13 release notes"
+menuTitle: "V2.13 release notes"
+description: "Release notes for Grafana Mimir version 2.13"
+weight: 1100
+---
+
+# Grafana Mimir version 2.13 release notes
+
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.13 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Store-gateway can be configured to explicitly disable or enable tenants**. This is useful in cases when compaction slows down for a tenant to temporarily exclude a tenant while compaction catches up without impact other tenants.
+
+- Configurable S3 bucket lookup type improves interoperability with **S3-compatible providers such as **Tencent and Alibaba\*\*\*\*. Configure lookup type via `-<prefix>.s3.bucket-lookup-type` or `-common.storage.s3-bucket-lookup-type`.
+
+- **Mimirtool can now verify the validity of a Mimir runtime configuration** file with the `mimirtool runtime-config verify` command.
+
+- **Configuration of TLS for S3 buckets** is now possible via `-<prefix>.s3.http.*` and `-common.storage.s3.http.*` configuration options.
+
+- **Improved OTLP ingestion performance** by using native translation, which reduces memory usage in the distributor by up to 30% and CPU usage by up to 8%.
+
+- **Active series are now updated along with owned series.** This means the number of active series for a tenant is much more accurate after adding ingesters.
+
+- **Remote read (`/prometheus/api/v1/read`) is becoming a first-class endpoint in Mimir** with support in `query stats` logs and query blocking in the query-frontend. More coming in future releases.
+
+- **Improved CPU performance in processing queries with regular expressions** that contain many alternations (for example `foo|bar|baz|...`).
+
+Additionally, the following previously experimental features are now considered stable:
+
+- Rules tenant federation via the `source_tenants` field in rule groups.
+- Enabling and recording and alerting rules evaluation on a per-tenant basis.
+- Limiting the number of tenants a federated query can query.
+
+## Important changes
+
+In Grafana Mimir 2.13 the following behavior has changed:
+
+- The default Docker image `grafana/mimir` is now based on the distroless image `gcr.io/distroless/static-debian12`.
+  See [Debugging distroless container images](https://grafana.com/docs/mimir/next/manage/mimir-runbooks/#debugging-distroless-container-images-in-kubernetes) for more details on how to
```

Comment:
Should this point to a 2.13-specific URL, rather than `next`?

Reply:
i wasn't sure what we want. `next` is probably not it. `latest` is more likely to give accurate results in future releases, but is also more likely to break as pages are reordered. `v2.13.x` won't break, but the information can become outdated. 

@tacole02 can you take a look here? This is a link to a section only published with the 2.13 release. We aren't sure what versioned link to use for it.

Reply:
Will the section be included in future releases that follow 2.13? If yes, then I think we should go with `latest`.

If this is only for 2.13 and won't go into future releases, we should specify 2.13 in the URL.

Reply:
Thanks! it will be included in future releases, so I'll just use `latest`

```suggestion
  See [Debugging distroless container images](https://grafana.com/docs/mimir/latest/manage/mimir-runbooks/#debugging-distroless-container-images-in-kubernetes) for more details on how to
```